arcgis-ios-sdk-samples/Content Display Logic/Controllers/SampleInfoViewController.swift

```diff
@@ -46,25 +46,28 @@ class SampleInfoViewController: UIViewController {
     
     func displayHTML(_ readmeContent:String) {
         let cssPath = Bundle.main.path(forResource: "style", ofType: "css") ?? ""
-        let string = "<!doctype html>" +
-        "<html>" +
-        "<head> <link rel=\"stylesheet\" href=\"\(cssPath)\">" +
-        "<link rel=\"stylesheet\" type=\"text/css\" href=\"https://cdnjs.cloudflare.com/ajax/libs/foundation/5.5.2/css/foundation.min.css\">" +
-        "<link rel=\"stylesheet\" type=\"text/css\" href=\"https://maxcdn.bootstrapcdn.com/font-awesome/4.4.0/css/font-awesome.min.css\">" +
-        "<meta name=\"viewport\" content=\"initial-scale=1, width=device-width, height=device-height, viewport-fit=cover\">" +
-        "</head>" +
-        " <div id=\"preview\" sd-model-to-html=\"text\">" +
-        "<div id=\"content\">" +
-        "\(readmeContent)" +
-        "</div></div>" +
-        "<script src=\"https://cdnjs.cloudflare.com/ajax/libs/showdown/1.1.0/showdown.js\"></script>" +
-        "<script>" +
-        "var conv = new showdown.Converter();" +
-        "var txt = document.getElementById('content').innerHTML;" +
-        "document.getElementById('content').innerHTML = conv.makeHtml(txt);" +
-        "</script>" +
-        "</body>" +
-        "</html>"
+        let string = """
+            <!doctype html>
+            <html>
+            <head>
+                <link rel="stylesheet" href="\(cssPath)">
+                <link rel="stylesheet" type="text/css" href="https://cdnjs.cloudflare.com/ajax/libs/foundation/5.5.2/css/foundation.min.css">
+                <link rel="stylesheet" type="text/css" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.4.0/css/font-awesome.min.css">
+                <meta name="viewport" content="initial-scale=1, width=device-width, height=device-height, viewport-fit=cover">
+            </head>
+            </body>
```

Comment:
shouldn't this be `<body>`?  i.e., no `/`

Reply:
🤦🏼‍♂️ Good catch. I copied and pasted the closing tag, forgetting to remove the slash.